spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -1144,6 +1146,8 @@ public synchronized void testDeleteWithSnapshotIsolation()
     sql(
         "ALTER TABLE %s SET TBLPROPERTIES('%s' '%s')",
         tableName, DELETE_ISOLATION_LEVEL, "snapshot");
+    sql("ALTER TABLE %s SET TBLPROPERTIES('%s' '%s')", tableName, COMMIT_MIN_RETRY_WAIT_MS, "10");
```

Comment:
no need to issue multiple ALTER statements. These properties can be set with one ALTER cmd

Reply:
Yes, I just think it will be easier to read.

Reply:
I don't think we want to issue X ALTER statements just for readability as it increases test duration. This should all happen in a single ALTER stmt